experiment/references.md

```diff
@@ -1 +1,17 @@
-### Link your references in here
\ No newline at end of file
+1. http://en.wikipedia.org/wiki/Surfactant
+
+2. http://nsdl.niscair.res.in/bitstream/123456789/311/1/PDF+corrected+surface+chemistry.pdf
+
+3. http://www.elmhurst.edu/~chm/vchembook/554soap.html
```

Comment:
Please use https references only and ensure that all of them work. Also provide meaningful link descriptions in place of putting the complete link there